hgtv/__init__.py

```diff
@@ -16,9 +15,6 @@
 app = Flask(__name__, instance_relative_config=True)
 lastuser = Lastuser()
 
-assets['hgtv.css'][version] = 'css/app.css'
```

Comment:
@vidya-ram ^

Reply:
There's a significance to using Flask-Assets to specify an asset, even if it's not processed in any way: it adds a version tag to the final URL, helping bust stale caches.

However, since it does this at run-time, if we run multiple servers, it will generate different version tags on each. This will cause cache duplication in the client, making it much harder to bust cache in a future deployment. Therefore, version tags need to be inserted at the build stage instead of run-time.

Reply:
Moved it here https://github.com/hasgeek/hasgeek.tv/pull/177/files#diff-844a936b5b1842a3d7811dc5ff7a64fe
So, that app related JS & CSS files are all inside assets folder and part of webpack build process.  Webpack service worker plugin, takes care of generating `service-worker.js` which in turn adds a hash to end of each of these asset files that help in cache bursting on the client side. Each build would re-calculate hashes and re-generate a new service worker.